lms/static/scripts/frontend_apps/components/LMSFilePicker.js

```diff
@@ -233,8 +220,21 @@ export default function LMSFilePicker({
         />
       );
       break;
+    case 'reload':
+      continueButton = (
+        <Button
+          onClick={() => {
+            setDialogState({
+              ...INITIAL_DIALOG_STATE,
+              continueAction: 'reload',
+            });
+            setTimeout(fetchFiles, 2000);
```

Comment:
Why is there a delay here? If there was a need for one, we should take care that it is cleared if any change (eg. component being unmounted) means it should no longer be triggered.